tap_s3_csv/csv_iterator.py

```diff
@@ -28,12 +26,6 @@ def get_row_iterator(iterable, options=None):
     reader.fieldnames = handle_empty_fieldnames(
         reader.fieldnames, options)
 
-    # csv.DictReader skips empty rows, but we wish to keep empty rows for csv imports, so override __next__ method.
-    # Only modifying for imports from csv connector for now as imports from s3 connector might have reasons for skipping empty rows.
-    # Could look into using csv.reader instead for cleaner code if s3 connector could also keep empty rows.
-    if options.get('is_csv_connector_import', False):
-        csv.DictReader.__next__ = next_without_skip
```

Comment:
This is no longer an issue. Both original csv.DictReader.__next__ and next_without_skip will skip empty lines (no commas), but keep lines with commas.